### DISCUSSION TITLE
Is support over pytorch Bert text classification available?

### DISCUSSION THREAD

**Discussion by smi45** [bentoml/BentoML#1123] [General]
HI,

For my use case i am using transformers "bert-base-german-cased" model and looking out for support from BentoML framework. I could not find any projects from the gallery in same line.

Could anyone help me on this?

**Comment by parano** [comment]
Hi @smi45, transformers support is coming out in the next release 0.9.0, here's some example usage: https://github.com/bentoml/BentoML/blob/master/bentoml/frameworks/transformers.py#L39

Is this what you are looking for?

**Comment by smi45** [comment]
Thanks @parano for the response. I am looking out for binary text classification using bert and transformers.
